kyo-core/jvm/src/test/scala/kyo/PathTest.scala

```diff
@@ -90,7 +90,7 @@ class PathTest extends Test:
                 _ <- useFile(first, text)
                 _ <- useFile(second, text)
                 v <- root.readAll("txt")
-            yield assert(v == List(first -> text, second -> text))
+            yield assert(v.toSet == Set(first -> text, second -> text))
```

Comment:
@fwbrasil this has been breaking in CI -- are you okay with the fix? I'm not sure what the behavior should be.

Reply:
hmm I don't see anything on our side that would shuffle the data coming from the Java file APIs. I think it's ok to mirror the behavior of the underlying implementation. I'll push the fix to main to avoid issues in PRs.